lib/fluent/plugin/in_tail.rb

```diff
@@ -252,6 +254,7 @@ def flush_buffer(tw)
                   else
                     @tag
                   end
+            record[@path_key] ||= tw.path unless @path_key.nil?
```

Comment:
I mean that `path_key` is not overwritten if record already has it.
This was the specification of the original implementation (https://github.com/fluent/fluentd/pull/281), but we can change the spec now. Do you prefer to overwrite?